packages/react/src/components/List/HierarchyList/HierarchyList.test.e2e.jsx

```diff
@@ -0,0 +1,208 @@
+        'Freddie Freeman': '1B',
+        'Josh Donaldson': '3B',
+        'Nick Markakis': 'RF',
+        'Austin Riley': 'LF',
+        'Brian McCann': 'C',
+        'Ozzie Albies': '2B',
+        'Kevin Gausman': 'P',
+      },
+      'New York Mets': {
+        'Jeff McNeil': '3B',
+        'Amed Rosario': 'SS',
+        'Michael Conforto is a super duper long name that will get cut off': 'RF',
+        'Pete Alonso': '1B',
+        'Wilson Ramos': 'C',
+        'Robinson Cano': '2B',
+        'JD Davis': 'LF',
+        'Brandon Nimmo': 'CF',
+        'Jacob Degrom': 'P',
+      },
+      'Washington Nationals': {
+        'Trea Turner': 'SS',
+        'Adam Eaton': 'RF',
+        'Anthony Rendon': '3B',
+        'Juan Soto': 'LF',
+        'Howie Kendrick': '2B',
+        'Ryan Zimmerman': '1B',
+        'Yian Gomes': 'C',
+        'Victor Robles': 'CF',
+        'Max Scherzer': 'P',
+      },
+    },
+  },
+};
+
+describe('HierarchyList', () => {
+  it('handles drag and drop with no selections', () => {
+    const onSelect = cy.stub();
+    const HierarchyListWithReorder = () => {
+      const [items, setItems] = useState([
+        ...Object.keys(sampleHierarchy.MLB['American League']).map((team) => ({
+          id: team,
+          isCategory: true,
+          content: {
+            value: team,
+          },
+          children: Object.keys(sampleHierarchy.MLB['American League'][team]).map((player) => ({
+            id: `${team}_${player}`,
+            content: {
+              value: player,
+            },
+            isSelectable: true,
+          })),
+        })),
+        ...Object.keys(sampleHierarchy.MLB['National League']).map((team) => ({
+          id: team,
+          isCategory: true,
+          content: {
+            value: team,
+          },
+          children: Object.keys(sampleHierarchy.MLB['National League'][team]).map((player) => ({
+            id: `${team}_${player}`,
+            content: {
+              value: player,
+            },
+            isSelectable: true,
+          })),
+        })),
+      ]);
+
+      return (
+        <div style={{ width: 400, height: 600 }}>
+          <HierarchyList
+            title="MLB Expanded List"
+            items={items}
+            editingStyle="single-nesting"
+            pageSize="lg"
+            isLoading={false}
+            isLargeRow={false}
+            onListUpdated={(updatedItems) => {
+              setItems(updatedItems);
+            }}
+            hasSearch
+            hasDeselection
+            onSelect={onSelect}
+            defaultExpandedIds={['New York Mets']}
+          />
+        </div>
+      );
+    };
+
+    mount(<HierarchyListWithReorder />);
+    cy.get('.iot--list-item').eq(8).find('[title]').should('have.text', 'Pete Alonso');
+    cy.get('.iot--list-item').eq(6).find('[title]').should('have.text', 'Amed Rosario');
+    cy.get(':nth-child(9) > [draggable="true"]').drag(':nth-child(7) > [draggable="true"]');
```

Comment:
Also find this difficult to read, can we make it more readable?

Reply:
I'll check, but in my previous experience cypress selectors _suck_. It uses a similar selector engine to jQuery under the hood, so not nearly as neat and tidy as what we're used to (spoiled?) with in testing-library.

Reply:
can't do anything about the selectors, but did add comments on what is expected. Also removed a bunch of the teams/players to pair it down a bit.

Reply:
I think David added `testing-library/cypress`

Reply:
Yeah, that doesn't really help here. I was able to add it in one spot, but otherwise, we're still forced to `.eq` and selectors like that to ensure that the people are in the correct location.